src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -32,73 +33,85 @@
 @Component
 public class SubscriptionSyncController {
   private final SubscriptionRepository subscriptionRepository;
+  private final SubscriptionService subscriptionService;
   private final ApplicationClock clock;
+  private final CapacityReconciliationController capacityReconciliationController;
 
   public SubscriptionSyncController(
-      SubscriptionRepository subscriptionRepository, ApplicationClock clock) {
+      SubscriptionRepository subscriptionRepository,
+      ApplicationClock clock,
+      SubscriptionService subscriptionService,
+      CapacityReconciliationController capacityReconciliationController) {
     this.subscriptionRepository = subscriptionRepository;
+    this.subscriptionService = subscriptionService;
+    this.capacityReconciliationController = capacityReconciliationController;
     this.clock = clock;
   }
 
   @Transactional
   public void syncSubscription(Subscription subscription) {
-    final Optional<org.candlepin.subscriptions.db.model.Subscription> entity =
+
+    // TODO: add metrics for subscriptions created and updated //NOSONAR
+    final Optional<org.candlepin.subscriptions.db.model.Subscription> maybePresent =
```

Comment:
I prefer variables names like `subscriptionOpt` so when you're reading further down (like line 60ish), `subscriptionOpt.isPresent()` is a bit more descriptive than `maybePresent.isPresent()`

Reply:
Makes sense. Renamed.